docs/administration/Monitor_and_Alert.md

```diff
@@ -8,75 +8,75 @@ The available metrics are:
 
 |Indicator|Unit|Type|Meaning|
 |---|:---:|:---:|---|
-|be_broker_count|pcs|average|Number of brokers |
-|be_brpc_endpoint_count|pcs|average|Number of StubCache in BRPC|
+|be_broker_count|count|average|Number of brokers |
```

Comment:
can we unify the format, whether need to capitalize the first letter? I saw some are bytes/s, some are Byte, some are Ms.

Reply:
use the lower case for now. and we will update the metrics in these two months

Reply:
use the lower case for now. and we will update the metrics in these two months

> can we unify the format, whether need to capitalize the first letter? I saw some are bytes/s, some are Byte, some are Ms.

use the lower case for now. and we will update the metrics in these two months